content/news/028/index.md

```diff
@@ -84,6 +84,44 @@ out [this form][gamedev-meetup-form].
 
 ## Game Updates
 
+### Shroom Kingdom
+
+![Shroom Kingdom game preview](shroom-kingdom.gif)
+_Starting the game and placing some blocks_
+
+Shroom Kingdom ([GitHub][shrm-github], [Discord][shrm-discord],
+[Twitter][shrm-twitter]) is an upcoming play-to-earn video game built with web
+technologies running on the [NEAR Blockchain][near-blockchain].
+
+Currently a prototype of the game is in development.
+An asset file, that has been extracted from Super Mario Maker 2 game files with an
+[external program](asset-extractor), is loaded inside a browser to start the game.
```

Comment:
I'm curious, are there any copyright issues with the extraction?

Reply:
The asset extractor is using an open source tool for decryption and unpacking: https://github.com/SciresM/hactool
This tool can extract most of the game files to a container format, which can then be parsed with e.g. this: https://github.com/tarnadas/ninres-rs
And then I just convert the images to png.

The user needs to provide the console keys and the game files for decryption, so it should be fine.
SciresM is also the developer of atmosphere, a CFW for Nintendo Switch and hactool is probably used a lot as a library in several applications.

The container formats that I parse also have various different libraries in different languages, e.g. this one: https://github.com/KillzXGaming/Switch-Toolbox

No copyrighted assets are bundled in the app nor do we share any of them